compiler/passes/scopeResolve.cpp

```diff
@@ -2741,6 +2746,114 @@ static void detectUserDefinedBorrowMethods() {
   }
 }
 
+
+/* Look up the symbol named 'name' and add it to the map and set of visible
+   symbols in a given file. The map maps from a filename to a set of visible
+   symbols.
+ */
+static void lookupAndAddToVisibleMap(const char* name, CallExpr* call,
+  std::map<std::string, std::set<Symbol*>*>& visibleMap,
+  std::set<Symbol*>& alreadyFound) {
+
+  int numSymbolsFound;
+  Symbol* found = lookupAndCount(name, call, numSymbolsFound,
+                                 false, NULL, false);
+  if (found != NULL && alreadyFound.count(found) == 0 &&
+      !found->hasFlag(FLAG_GLOBAL_VAR_BUILTIN) &&
+      !found->hasFlag(FLAG_COMPILER_GENERATED) &&
+      !found->hasFlag(FLAG_TEMP) &&
+      strcmp(found->defPoint->fname(), "<internal>") != 0) {
+    const char* fname = found->defPoint->fname();
+    alreadyFound.insert(found);
+    if (visibleMap.count(fname) == 0) {
+      visibleMap.insert({fname, new std::set<Symbol*>()});
+    }
+    visibleMap[(std::string)fname]->insert(found);
+  }
+}
+
+
+/* Find any "get visible symbols" primitive calls and print out all
+   symbols that are visible from that point.
+ */
+static void processGetVisibleSymbols() {
+  forv_Vec(CallExpr, call, gCallExprs) {
+    if (call->isPrimitive(PRIM_GET_VISIBLE_SYMBOLS)) {
+      bool ignoreInternalModules = false;
```

Comment:
I'd suggest making this true by default.  I don't think that either we, or most users, will want to see the internal modules most of the time (until / unless we're reviewing all of the internal modules).  Alternatively, we could have it take its default based on CHPL_DEVELOPER (though that may be more confusing to developers than beneficial).

Reply:
I just changed the default to true.